opencti-platform/opencti-graphql/src/modules/exclusionList/exclusionList-graphql.ts

```diff
@@ -0,0 +1,8 @@
+import { registerGraphqlSchema } from '../../graphql/schema';
+import exclusionListTypeDefs from './exclusionList.graphql';
+import exclusionListResolver from './exclusionList-resolver';
+
+registerGraphqlSchema({
```

Comment:
don't know if we can the whole domain and resolver under FF?

Reply:
domain function and elastic model are under FF, it should be enough :)

Reply:
discussed with Julien *